frontend/appflowy_flutter/packages/flowy_svg/lib/src/flowy_svg.dart

```diff
@@ -74,12 +74,15 @@ class FlowySvg extends StatelessWidget {
   /// The opacity of the svg
   ///
   /// The default value is 1.0
-  final double opacity;
+  final double? opacity;
 
   @override
   Widget build(BuildContext context) {
-    final iconColor =
-        (color ?? Theme.of(context).iconTheme.color)?.withOpacity(opacity);
+    Color? iconColor = color ?? Theme.of(context).iconTheme.color;
+    if (opacity != null && iconColor != null) {
```

Comment:
Why not just `if (opacity != null) {`?

Reply:
iconColor is still nullable here so we still have to check for null. I guess it's a bit cleaner?

```
  Color? iconColor = color ?? Theme.of(context).iconTheme.color;
  if (opacity != null) {
    iconColor = iconColor?.withOpacity(opacity!);
  }
```